pkg/activator/net/throttler.go

```diff
@@ -524,3 +526,23 @@ func (ib *InfiniteBreaker) Maybe(ctx context.Context, thunk func()) error {
 		return ctx.Err()
 	}
 }
+
+// SetCapacity is a test helper that sets Throttler in a desired
+// state for unit testing external components.
+// This permits us to hide behind the veneer the details of how Thorttler is updated.
+func (t *Throttler) SetCapacity(ns, n string, backends int, clusterIP string) {
+	upd := revisionDestsUpdate{
+		Rev:           types.NamespacedName{ns, n},
+		ClusterIPDest: clusterIP,
+		Dests:         dests(backends),
+	}
+	t.handleUpdate(upd)
+}
+
+func dests(count int) sets.String {
```

Comment:
Same for moving to a `_test` file.

Reply:
And same, and probably I can inline it since it is for tests so it's ok